.gitlab-ci.yml

```diff
@@ -294,3 +303,33 @@ publish-docs:
     - git push origin gh-pages --force
   after_script:
     - rm -rf .git/ ./*
+
+#### stage:                        fuzz
+
+fuzz-tests:
+    stage:                           fuzz
+    <<:                              *docker-env
+    variables:
+        # The QUICKCHECK_TESTS default is 100
+        QUICKCHECK_TESTS:            50000
```

Comment:
I'd personally prefer a solution that simply has a timeout of approx. 10 minutes or so but for a start this should suffice our needs as well. :)